Sources/PropertyWrappers.swift

```diff
@@ -36,30 +36,32 @@ public struct SwiftyUserDefaultOptions: OptionSet {
 }
 
 @propertyWrapper
-public final class SwiftyUserDefault<T: DefaultsSerializable> where T.T == T {
+public final class SwiftyUserDefault<T: DefaultsSerializable, KeyStore: DefaultsKeyStore> where T.T == T {
 
     public let key: DefaultsKey<T>
     public let options: SwiftyUserDefaultOptions
+    private var adapter: DefaultsAdapter<KeyStore>
 
     public var wrappedValue: T {
         get {
             if options.contains(.cached) {
-                return _value ?? Defaults[key: key]
+                return _value ?? adapter[key: key]
             } else {
-                return Defaults[key: key]
+                return adapter[key: key]
             }
         }
         set {
             _value = newValue
-            Defaults[key: key] = newValue
+            adapter[key: key] = newValue
         }
     }
 
     private var _value: T.T?
     private var observation: DefaultsDisposable?
 
-    public init<KeyStore>(keyPath: KeyPath<KeyStore, DefaultsKey<T>>, adapter: DefaultsAdapter<KeyStore>, options: SwiftyUserDefaultOptions = []) {
+    public init(keyPath: KeyPath<KeyStore, DefaultsKey<T>>, adapter: DefaultsAdapter<KeyStore>, options: SwiftyUserDefaultOptions = []) {
```

Comment:
so ideally I'd love to have the `Defaults` be a default adapter - can we do that?

Reply:
So I was thinking we would be able to utilize a global framework property as a default initializer but that doesn't appear to be the case. I might suggest that we set default initializer with the same adapter init used for `Default` for this public init. This implies that unless a different adapter is provided, it will always look for key/values in the `standard` UserDefaults using the `DefaultsKeyStore`. Which may be practical for most basic use cases.

Reply:
@spydercapriani yeah that would work - thanks!